guides/doc-Content_Management_Guide/topics/Managing_OSTree_Branches.adoc

```diff
@@ -8,6 +8,17 @@ Red Hat Enterprise Linux Atomic Server uses OSTree content composed from RPM fil
 
 You can use {ProjectNameX} to synchronize and manage OSTree branches from an OSTree repository.
 
+[NOTE]
+====
+The following management operations of OSTree and Puppet content types are deprecated and will be removed in the next release:
+
+* Creating OSTree and puppet repositories.
+* Synchronizing OSTree and puppet repositories.
+* Adding repositories of OSTree and puppet content types to content view.
+* Publishing and promoting the content views containing repositories of OSTree and puppet content types across life cycle environments.
+
+====
```

Comment:
You can remove this entire note as all of this content has been removed. It was deprecated in the previous release. 
This should be added release notes about removals.

Reply:
Changed as per comment.